targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L496xG/device/TOOLCHAIN_IAR/stm32l496xx_flash.icf

```diff
@@ -0,0 +1,68 @@
+/*###ICF### Section handled by ICF editor, don't touch! ****/
+/*-Editor annotation file-*/
+/* IcfEditorFile="$TOOLKIT_DIR$\config\ide\IcfEditor\cortex_v1_0.xml" */
+/*-Specials-*/
+define symbol __ICFEDIT_intvec_start__ = 0x08000000;
+/*-Memory Regions-*/
+define symbol __ICFEDIT_region_ROM_start__        = 0x08000000;
+define symbol __ICFEDIT_region_ROM_end__          = 0x0807FFFF;
+define symbol __ICFEDIT_region_ROM_TARA_start__   = 0x08080000;           /* TARA Lite Demo */

```

Comment:
Thank you for your review !
I get this file on STM32Cube as indicated here [ST porting guide](https://developer.mbed.org/teams/ST/wiki/steps-to-create-a-new-STM32-platform) but it seems that I took it from a project example. I will replace it with the template provided by STM32Cube.